### PR TITLE
rename rgba conversions

### PR DESCRIPTION
**Pull request by theomonnom**
So we're not following FourCC as it is confusing for users?
But instead we follow the logic from OpenCV etc

### CODE REVIEW

**Review by keepingitneil — APPROVED** [review]

File: livekit-ffi/src/server/requests.rs [hunk 1]
```diff
@@ -470,19 +472,19 @@ unsafe fn on_to_argb(
             match rgba_format {
                 proto::VideoFormatType::FormatArgb => {
                     #[rustfmt::skip]
-                    yuv_helper::i420_to_argb(src_y, yuv.stride_y, src_u, yuv.stride_u, src_v, yuv.stride_v, argb, stride, w, h);
+                    yuv_helper::i420_to_bgra(src_y, yuv.stride_y, src_u, yuv.stride_u, src_v, yuv.stride_v, argb, stride, w, h);
```
Comment (davidzhao) [inline livekit-ffi/src/server/requests.rs:475]: I'm not understanding this, isn't this mismatched from the indicated format?

Comment (theomonnom) [reply livekit-ffi/src/server/requests.rs:475]: FourCC are defined in BigEndian https://en.wikipedia.org/wiki/FourCC#Compiler_support (It is just the naming convention)
So on little endian system ARGB is indeed BGRA in memory

Comment (theomonnom) [reply livekit-ffi/src/server/requests.rs:475]: Also more details here https://chromium.googlesource.com/libyuv/libyuv/+/refs/heads/main/docs/formats.md#the-argb-fourcc

Comment (davidzhao) [reply livekit-ffi/src/server/requests.rs:475]: ah got it. thanks!

**Review by davidzhao — APPROVED** [review]

**Pull request merged**